libs/src/api/account.js

```diff
@@ -89,10 +91,10 @@ class Account extends Base {
    * clients may wish to call window.location.reload()
    * to show the user as logged out
    */
-  logout () {
+  async logout () {
     if (!this.web3Manager.web3IsExternal()) {
       this.REQUIRES(Services.HEDGEHOG)
-      this.hedgehog.logout()
+      await this.hedgehog.logout()
```

Comment:
All of the changes in this file were from running the linter except for the changes to this method

Reply:
We really need to do a lint pass on all of libs at some point